widgetry/src/text.rs

```diff
@@ -69,6 +69,12 @@ impl TextSpan {
         }
         self
     }
+    pub fn size(mut self, size: usize) -> TextSpan {
```

Comment:
I don't think we should add this to the API yet. Originally, font family and size could be set manually like this, but over time, the text style became very inconsistent everywhere. We consolidated on a few styles to apply throughout the UI. Over in `info/trip` you're using this to scale down the zoomed-in text. You could instead just do `.scale(0.1)` after `render(ctx)`, tuning the `0.1`.

Reply:
I didn't realise about the scale, and couldn't find a way to reduce font size.
But yeah, can definitely change it.

Reply:
Removed